### PR TITLE
Made a button with react and mounted it.

### PR DESCRIPTION
**Pull request by K-Karneboge**
image

### CODE REVIEW

File: react-function-components/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: The import for `ReactDOM` should be:
```JAVASCRIPT
import ReactDOM from 'react-dom/client`;
```

File: react-function-components/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+const CustomButton = <button>Click Me!</button>;
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: Here we should be using a function to create a function component:
```JSX
function CustomButton() {
  return <button>Click Me!</button>;
}
```

File: react-function-components/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+const container = document.querySelector('#root');
+
+const root = ReactDOM.createRoot(container);
+
+root.render(CustomButton);
```
Comment (scbowler) [inline react-function-components/src/index.jsx]: To use a component we use JSX:
```JSX
root.render(<CustomButton />);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
A few updates needed. Let me know if you have any questions.

**Comment by K-Karneboge** [comment]
image
Changed, the index html in dist didnt seem to differ, however.

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**